source/browseMode.py

```diff
@@ -2132,13 +2132,14 @@ def _handleScrollTo(
 		else:
 			raise ValueError(f"{obj} is not a supported type")
 
-		# We only want to update the caret and speak the field if we're not in the same one as before
+		# We only want to update the caret and speak the field if we're not in the first line of the same object as before.
+		# See https://github.com/nvaccess/nvda/issues/17669
```

Comment:
I think we tend to do just issue numbers for internal issues

```suggestion
		# See #17669
```